.github/workflow_templates/deploy-channel.multi.yml

```diff
@@ -35,6 +35,9 @@ on:
       editions:
         description: 'Comma separated editions to deploy. Example: ee,fe,ce'
         required: false
+      cooldown:
+        description: 'Postpone release until specified datetime (RFC3339). Example: 2026-06-06T16:16:16Z'
```

Comment:
I think it looks inconvenient to use (T and Z must be specified).

Maybe simplify the format? E.g. YYYY-MM-DD HH:MM ?

Reply:
Done. We can use any of these formats:
- `YYYY-MM-DD HH:MM`
- `YYYY-MM-DD HH:MM:SS`
- `RFC3339`

everything is supported

Reply:
`YYYY-MM-DD HH:MM` is set as default and put into docs